webapp/src/ts/reducers/messages.ts

```diff
@@ -0,0 +1,30 @@
+import { createReducer, on } from '@ngrx/store';
+
+import { Actions } from '../actions/messages';
+
+export interface MessagesState {
+  error: boolean;
+  conversations: object[],
```

Comment:
I'll add proper typing in next PR when loading message content and have better idea of the fields.

Reply:
Remember to only do the bare minimum to replace AngularJS - we can fix typing and other improvements in a future release.